console/console-init/ui/src/utils/utils.ts

```diff
@@ -173,6 +173,7 @@ const getLabelForTypeOfObject = (value: any) => {
     case "boolean":
       return "Boolean";
   }
+  return "";
```

Comment:
why do we need this return statement here.we don't need it.

Reply:
![Screenshot from 2020-09-09 02-46-24](https://user-images.githubusercontent.com/66817941/92528753-e9a8ae80-f246-11ea-97c7-c2891fec062a.png)

If we don't add the empty return statement the following error occurs. It does not accept an undefined string.

Reply:
ajaypratap003 actually just to be type safe IMetadataProps is expecting type to be string and the method return type is string|undefined. I don't see any case where the function will return empty string. That's why in case of undefined it will return  a empty string

Reply:
ok